css/component.css

```diff
@@ -0,0 +1,34 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap");
+@import url("/css/style.css");
+:root {
+	--COLOR-PRIMARY: #01ccff;
+	--COLOR-SECONDARY: #27e4b5;
+	--COLOR-SOCIAL-ICON: #308ea5;
+	--PRIMARY-HOVER: #7a9ea7;
+	--NOTIFICATION-COLOR: #7cf916;
+	--offgreen: #bacfc0;
+	--darkerofseconadary: #2195d8;
+	--BOX-SHADOW: rgb(175, 152, 152);
+	--BOX-SHADOW-LIGHTER: rgb(78, 76, 37);
+	--ALERT-PRIMARY-COLOR: #2563eb;
+	--ALERT-SUCCESS-COLOR: rgb(13, 224, 31);
```

Comment:
Suggestion - Try to use one code value for color.

Reply:
+1 this change is pending